src/main/java/org/polyfill/configurations/AppConfig.java

```diff
@@ -0,0 +1,27 @@
+package org.polyfill.configurations;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+/**
+ * Created by reinier.guerra on 1/24/17.
+ */
+@Configuration
+public class AppConfig {
```

Comment:
This is one way to store string constants. It allows us to swap the strings during testing. I don't like this though since it requires so many lines just for those paths. We should find a way to store the strings nicely and also good for testing.